script/installation/packages.sh

```diff
@@ -78,7 +78,8 @@ elif [[ "$DISTRO" == *"FEDORA"* ]]; then
         llvm-devel \
         llvm-static \
         libedit-devel \
-        postgresql
+        postgresql \
+        libatomic
```

Comment:
Since this is due to GCC 7 that calls are routed to libatomic, I guess we need to install libatomic on  Ubuntu 17.10 as well which has GCC 7. I don't have any Ubuntu installation around and from a quick search couldn't find the name of the package that provides libatomic.so there.

But worth checking if it's *really* required on Ubuntu 17.10. Maybe Ubuntu packagers did some magic not requiring to link against libatomic at all :)